packages/bathymetry/src/bathy.maker.ts

```diff
@@ -107,15 +117,11 @@ export class BathyMaker {
     }
 
     /** Render a specific tile */
-    async renderTile(tile: Tile, logger: LogType): Promise<void> {
-        try {
```

Comment:
This logger has a log context including the tile index that failed.

if you want to remove the log line here, maybe catch the error and re throw a composite error with which tile failed